MIDAS/src/hardware/Camera.cpp

```diff
@@ -0,0 +1,60 @@
+#include <ACAN2517.h>
+#include <SPI.h>
+
+static const byte MCP2517_SCK  = 0;
```

Comment:
if you rebase/merge in mvp_sensor_integration, there is a pins.h file that defines pins for the general spi channel, so you can move these into that file. also, we want pins to be macros